bin/duh-export-scala.js

```diff
@@ -28,11 +28,27 @@ const argv = yargs
   .help()
   .argv;
 
+const fixupPortDirections = duh => new Promise((resolve) => {
+
+  // expand ports
```

Comment:
This is not quite "expanding" the ports. It's more like resolving any conflicts between the analog and raw wire direction.

Reply:
oops copy-paste error